packages/frontend/app/services/connection/connection.ts

```diff
@@ -0,0 +1,120 @@
+import { Channel, Socket } from 'phoenix';
```

Comment:
More and more code is going to end up being native JavaScript, as the "domain" has no impact on Ember / the view.